source/Account.cpp

```diff
@@ -191,20 +197,40 @@ string Account::Step(int64_t assets, int64_t salaries)
 			++it;
 	}
 	
```

Comment:
Just something to consider: Does it make sense to pay the maintenance (and operating) costs before the mortgages?

Reply:
Well now that overdue maintenance impact your credit score, maintenance doesn't need to be last and might as well occur alongside crew costs since they act in the same way.